tests/test_default_project.py

```diff
@@ -0,0 +1,37 @@
+from typing import Any, Dict, Text
+
+from warnings import WarningMessage
+from _pytest.pytester import Testdir
+import pytest
+
+from rasa.__main__ import create_argument_parser
+import rasa.cli.data
+import rasa.cli.scaffold
+import rasa.cli.train
+import rasa.shared.utils.io
+
+
+def _warning_should_be_filtered_out(warning: WarningMessage) -> bool:
+    # we filter out `gelu` warnings because of this issue:
+    # https://github.com/RasaHQ/rasa/issues/9129
+    # this function can be removed once we migrate to TensorFlow 2.6
+    return type(warning.message) == DeprecationWarning and str(
+        warning.message
+    ).startswith("gelu activation has been migrated to core TensorFlow")
+
+
+@pytest.mark.timeout(300, func_only=True)
+def test_default_project_has_no_warnings(
+    testdir: Testdir, default_config: Dict[Text, Any]
+):
+    parser = create_argument_parser()
+    rasa.cli.scaffold.create_initial_project(".")
+    rasa.shared.utils.io.write_yaml(default_config, "config.yml")
+
+    with pytest.warns(None) as warning_recorder:
+        rasa.cli.data.validate_files(parser.parse_args(["data", "validate"]))
+        rasa.cli.train.run_training(parser.parse_args(["train"]))
```

Comment:
Should `rasa shell` also be added to the test since it loads the model?

Reply:
`rasa shell` will be running indefinitely just because it expects an input. In addition, I don't see benefits of doing that because loading a model cannot print warnings anyway — it's an info message and then an error if something went wrong.

Reply:
Makes sense 👍🏼